met-api/src/met_api/models/contact.py

```diff
@@ -58,3 +58,27 @@ def create_contact(cls, contact) -> Contact:
         db.session.commit()
 
         return new_contact
+    
+     
+    @classmethod
+    def update_contact(cls, contact_id, contact_dict) -> Optional[Contact]:
+        """Update contact."""
+        query = Contact.query.filter_by(id=contact_id)
+        contact: Contact = query.first()
+        if not contact:
+            return None
```

Comment:
if this is the case than something is wrong, we should throw an error saying the record was not found

Reply:
Was this resloved?